modules/op-release-notes-1-8.adoc

```diff
@@ -163,7 +163,7 @@ include::snippets/technology-preview.adoc[]
 :FeatureName: {tekton-hub}
 include::snippets/technology-preview.adoc[]
 
-* With this update, you can disable the login and rating features of your {tekton-hub} web console. To do so, do not add a value for the `AUTH_BASE_URL` field in the `<tekton_hub_repository>/config/03-ui/30-ui-configmap.yaml` file.
+* With a fresh installation of {pipelines-title} 1.8 and later, if you install {tekton-hub} manually, not by using the Operator, you can disable the login and rating features of your {tekton-hub} web console by removing the `AUTH_BASE_URL` key-value pair from the `tekton-hub-ui` configmap in the `openshift-pipelines` namespace. If you install the {tekton-hub} by using the Operator, the login is already disabled by default.
```

Comment:
Based on our slack thread, it seems like we might be able to simply remove this release note.

Reply:
Yeah we can remove this point and may be we can add some more information below this point https://github.com/openshift/openshift-docs/pull/49483/files#diff-70dafaae46d75da8c0c8b5f6e2a70bb5f0df5d5368ce9dafc9a7afb5025c6dafL136 as


**NOTE** :- With this update, since login is already enabled in the previous release i.e 1.7.x, with operator upgrade by default login will be enabled, the disable login feature will be enabled only if user installs a fresh Hub instance (i.e. user doesn't creates any secrets and configmap, it is all created by the operator)

Reply:
Updated here instead. Please review.